user_data/strategies/Strategy002.py

```diff
@@ -1,23 +1,19 @@
 
 # --- Do not remove these libs ---
-from freqtrade.persistence import Trade
-
-
-from freqtrade.strategy import IStrategy
 from pandas import DataFrame
 # --------------------------------
 
 import talib.abstract as ta
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 import numpy # noqa
-from datetime import datetime
 
-from wao.strategy_controller import StrategyController
+from wao.WAOStrategy import WAOStrategy
 
 
-class Strategy002(IStrategy):
-    controller = StrategyController('Freq_Strategy002')
+class Strategy002(WAOStrategy):
+    WAOStrategy.controller.setup('Freq_Strategy002')
```

Comment:
do not use static reference, these variables should be accessible in the child class, as long as they are properly declared in the parent

Reply:
so far the solution I found for WAOStrategy.controller.setup('Freq_Strategy002') was to use "def __init__" but since that is inherited from the IStrategy it creates another complication and the code crashes so what do you recommend I do

Reply:
please schedule a call